blackbird_python/blackbird/tests/test_listener.py

```diff
@@ -45,37 +46,32 @@ class DummyDeviceContext:
 
 
 @pytest.fixture
-def parser():
+def parse_input():
```

Comment:
How come one function is called `parse_...` and the other is `parser_...` (they both seem to parse?)

Reply:
typo, good catch. Fixed in the latest commit.